src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -31,12 +31,12 @@ internal class FunctionExecutor : IFunctionExecutor
         private readonly IAsyncCollector<FunctionInstanceLogEntry> _functionEventCollector;
         private readonly ILogger _logger;
         private readonly ILogger _resultsLogger;
-        private readonly IJobInvoker _jobInvoker;
+        private readonly IExtensionRegistry _extensionRegistry;
```

Comment:
No longer referencing extension registry in executor now. Passing in collection of global filters.